src/index.html

```diff
@@ -1,12 +1,449 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="main_body main_body--normalize">
+    <header class="main_body__header main_body__header--position">
+      <div class="logo logo--position"></div>
+      <nav>
+        <ul class="nav_container">
+          <li class="nav_container__list">
+            <a class="nav_container__element is-active">
+              <span>Apple</span>
```

Comment:
doo you really need these spans?

Reply:
it's not about span or div
it's about you need neither
I'd prefer to use after to create this underline, but not critical

Reply:
Understood. I remake this)